parliament/upgrade.js

```diff
@@ -7,16 +7,23 @@
 // 4 - remove parliament password
 // 5 - move settings to config
 // 6 - combine notifiers with viewer
+// 7 - remove parliament json
+
+const uuid = require('uuid').v4;
 
 const Notifier = require('../common/notifier');
 
-const version = 6;
+const version = 7;
 
 /**
  * Upgrades the parliament object to the latest version
  * @param {object} parliament the parliament object to upgrade
+ * @param {object} ArkimeConfig the ArkimeConfig object
+ * @param {string} parliamentName the name of the parliament (must be unique)
+ * @param {object} issues the issues object to update
+ * @param {object} Parliament the Parliament class
  */
-exports.upgrade = async function (parliament, ArkimeConfig) {
+exports.upgrade = async function (parliament, ArkimeConfig, parliamentName, issues, Parliament) {
```

Comment:
maybe don't pass in ArkimeConfig - there was an issue with it not being initialized properly, change the order of the import
parliamentName can come from Parliament.#name (make it accessible)
can read issues directly from json
Parliament and be imported from parliament.js?